deployments/setup-fork.ts

```diff
@@ -96,8 +111,32 @@ const fundAccounts = async () => {
         }
     ];
 
+    if (DEV_ADDRESSES === undefined || DEV_ADDRESSES === '') {
+        Logger.log('No dev addresses to fund');
+        return;
+    }
+
     const devAddresses = DEV_ADDRESSES.split(',');
 
+    for(const fundingRequest of fundingRequests) {
+        if(fundingRequest.token == ZERO_ADDRESS) {
+            Logger.log(`Skipping funding for ${fundingRequest.token}`);
+        }
+        const { whale } = fundingRequest;
+        if (!whale) {
+            continue;
+        }
+        const whaleBalance = await whale.getBalance();
+        // transfer ETH to the funding account if it doesn't have ETH
+
+        if (whaleBalance.lt(toWei(1))) {
+            await fundingRequests[0].whale.sendTransaction({
```

Comment:
You assuming that `fundingRequests[0]` is `ETH`, but it doesn't have to be the case. Perhaps you can explicitly `find` `ETH`?

Reply:
fixed